packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1329,8 +1402,23 @@ class RenderCustomPaint extends RenderProxyBox {
     RenderBox child
   }) : _painter = painter, _foregroundPainter = foregroundPainter, super(child);
 
+  /// The background custom paint delegate.
+  ///
+  /// This painter, if non-null, is invoked to paint behind the children.
```

Comment:
Suggest using active voice:
"If non-null, the framework invokes this method to paint behind the children (widgets?)."